lib/import/admin.js

```diff
@@ -142,17 +150,57 @@ function save_features(feature_collection) {
 }
 
 /**
+ * Get hash of admin to population
+ * mutates `feature_collection`.
+ *
+ * @param{string} country_code - Country Code
+ * @param{array} feature_collection - Feature collection
+ * @return{Promise} Fulfilled when csv is read.
+ */
+function world_pop(country_code, feature_collection) {
+  var path = './data/population/';
+  var file = 'worldpop_' + country_code + '.csv';
+  var stream = fs.createReadStream(path + file);
```

Comment:
`fast-csv` has a more convenient method of reading a file that doesn't require setting up streams manually. See the [.fromPath method](https://github.com/C2FO/fast-csv).